package.json

```diff
@@ -25,15 +25,21 @@
   },
   "homepage": "https://github.com/jst-seminar-rostlab-tum/truffle-ai-backend#readme",
   "dependencies": {
+    "@types/axios": "^0.14.0",
```

Comment:
Why there is two different implementations like @types/axios and axios

Reply:
I didn't mean to add them. Sorry